### ISSUE TITLE
This list is so out of date is borders on irresponsible

### ISSUE THREAD

**Issue by rwxrob** [dariubs/GoBooks#155]
I'm sure you will immediately close this, but books from 2018 have no business being on any "awesome" list. Most do not even cover modules, let alone generics, and `go work`. I had to open this issue because some person told me "why make a book when there are already tons of good books out there" and told me about this.

**Comment by Tranberry** [comment]
I believe listing the year the book was last revised is mostly "solving" this issue, it might be helpful to include a note about key breakpoints in Go's development, such as major features introduced in specific versions of Go.

The challenge, I assume, is moderation—it’s a time-intensive process. Plus, deciding which features are important is often (not always) subjective and could lead to conflicts that demand even more time to resolve.

**Comment by nathany** [comment]
> books from 2018 have no business being on any "awesome" list

As the author of one of those books from 2018, I'm not sure that I agree that my book is no longer awesome. 😅  Especially since all the code continues to compile in current Go versions.

Are there other books that are newer and probably a better choice to learn Go now, probably. But "awesome" is somewhat subjective depending on your needs. If you want to teach your preteen Go using the Go Playground, I think my book still holds up as a good starting point.

> "why make a book when there are already tons of good books out there"

IMO, there is always room for new books -- not just for more up-to-date coverage -- but also with your particular take on the exercises and topics covered.

> I believe listing the year the book was last revised is mostly "solving" this issue

I think so. Though I do wonder if we should reorder the books from newest to oldest?

**Comment by rwxrob** [comment]
Yeah, I would put most recent on the top. I would also call out the really bad books like Mastering Go that covered a proposal for generics only to be dead wrong.
